tailwind.config.js

```diff
@@ -81,8 +83,43 @@ module.exports = {
         neufile: "Neufile"
       },
       
+      width:{
+        2.785: "0.69625rem",
+        4.0725: "1.018125rem",
+        47.3375 : "11.8344rem",
+        66 : "16.5rem",
+        84.5 : "21.125rem",
+        141.25 : "35.3125rem",
+        153.5 : "38.375rem",
+
+      },
+
+      maxWidth:{
+        "maxWidth": "89.17%"
```

Comment:
Since we may have many `max-width` values, try to use numeric names instead of using normal names.

Reply:
done